README.md

```diff
@@ -1,50 +1,129 @@
 
-To learn more on how to set the `GOOGLE_APPLICATION_CREDENTIALS` variable refer this link https://cloud.google.com/docs/authentication/getting-started
+## ENVs and Secrets
 
-`SPACECRAFT_GCP_PROJECT` is the project id which you want to use for this tool and `SPACECRAFT_GCP_LOCATION`/`SPACECRAFT_GCP_ZONE` indicates the location and zone respectively for the GKE cluster.
+To use spacecraft you need to have the following ENVs set:
 
-### Clone and Build CLI
+```jsx
+# GCP Authentication
 
-Clone the repository and build the CLI binary. Run the below commands to do that:
+SPACECRAFT_GCP_PROJECT=
+SPACECRAFT_GCP_LOCATION=
+SPACECRAFT_GCP_ZONE=
+GOOGLE_APPLICATION_CREDENTIALS=
 
+# Cloudflare Authentication
+
+SPACECRAFT_CLOUDFLARE_API_TOKEN=
+
+# Slack Authentication
+
+SPACECRAFT_SLACK_CHANNEL_ID=
+SPACECRAFT_SLACK_TOKEN=
+
+# SSL Cert and Key
+
+SPACECRAFT_TLS_CRT=
+SPACECRAFT_TLS_KEY=
 ```
-git clone git@github.com:spacemeshos/go-spacecraft.git
-cd go-spacecraft
-go build
-```
 
-### Create the network
+If you don't want slack alerts or if you don't want to deploy web services then you can omit the slack and SSL ENVs respectively. 
+
+> All the secrets are stored in 1password and tagged as go-spacecraft.
+> 
+
+## Using the CLI
 
-The final step is to create the network. Run the below command to create a devnet:
+You can download and compile spacecraft using the following commands:
 
+```jsx
+git clone git@github.com:spacemeshos/go-spacecraft.git
+cd go-spacecraft && go build
 ```
-./go-spacecraft createNetwork --config=./artifacts/devnet/config.json
+
+Now you can explore the CLI flags and sub-commands using:
+
+```jsx
+./go-spacecraft --help
 ```
 
-This command will take around ~10min to finish and at the end you should see the kibana URL. The generated config file is uploaded to GCP storage. 
+Now you can refer to [devnet deployment guide](docs/devnet.md) to understand how to use the CLI to deploy a new network and web services.
+
+## Network Deployment
+
+For creating a new network boostrap deploys one bootstrap node, multiple bootnodes and then multiple miners. The bootnodes connect to bootstrap for p2p discovery and miners connect to bootnodes for p2p discovery. 
+
+Spacecraft also allows you to create a network that actually extends another network i.e., deploys a network without bootstrap. In this case the bootnodes connect to the bootnodes of the other network and miners of this new network connect to the bootnodes of this new network. This can be useful when you want to delete all the managed miners and other deployments and create a fresh network that connects to an existing network. You can achieve this using the `--bootstrap=false` flag.
+
+Whenever a network is created it also deploys a N number of PoETs. The PoET are assigned to the nodes in round robin fashion. The first node is then added to the config file so home smeshers use the first PoET always. 
```

Comment:
How are nodes being assigned to the PoETs, as their gateway nodes?

Reply:
Was this resolved? Still can't find info about it.

Reply:
Yes its resolved. I had added this line "The PoET are assigned to the nodes in round robin fashion using `poet-server` config"